src/components/PostContent.js

```diff
@@ -16,7 +17,7 @@ import ChatIcon from '@material-ui/icons/Chat';
 /** View component for displaying an individual post's content'
  * @param {IPostComponentProps} props
  */
```

Comment:
add props.post and props.classes to your type annotation here

Reply:
Fixed